### ISSUE TITLE
duplicate lines 30:31

### ISSUE THREAD

**Issue by mft3000** [josenk/terraform-provider-esxi#3]
Hi Josenk,

i'm working on a Dockerfile with terraform and your plug-in (you made a great job) and i'm hitting a syntax error

esxi/esxi_remote_cmds.go:30: syntax error: non-declaration statement outside function body

I think you duplicate two lines.

thank you for your time

Francesco

https://github.com/josenk/terraform-provider-esxi/blob/69185e2633045c848d5c0fc12671c747892fdf79/esxi/esxi_remote_cmds.go#L27

**Comment by josenk** [comment]
Yes, sorry about that...   It was from a merge request that wasn't properly formatted.  Should be fixed now in the Tagged v1.0.2 release.

**Comment by mft3000** [comment]
no problem at all... glad to helped you 👍
